pubspec.yaml

```diff
@@ -80,6 +80,21 @@ dependencies:
       url: git://github.com/linagora/jmap-dart-client.git
       ref: master
 
+  # http_parser
+  http_parser: 4.0.0
+
+  # file_size
+  filesize: 2.0.1
+
+  # flutter_widget_from_html
+  flutter_widget_from_html: 0.6.2
+
+  # photo_view
+  photo_view: 0.12.0
```

Comment:
use for what?

Reply:
click view photo in email content

Reply:
You did the work out of the scope :). Please focus to the scope of the ticket next time.